src/python/pants/backend/python/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
Why is this needed? Ditto the other `__init__.py` change below.

Reply:
For the coverage tests in Pants where the backend/python/ code here is in the gather sources pex and the backend/python/tasks/coverage plugin code is in the pytest-prep pex. Without a namespace package arrangement through backend/python/tasks the plugin can't be found by coverage.

Reply:
gotcha, makes sense.